ansys-grantami-serverapi-openapi/README.md

```diff
@@ -2,7 +2,7 @@
 
 Autogenerated client library for the Granta MI Server API.
 
-Direct use of this package is unsupported, please use [ansys-grantami-recordlists](https://github.com/ansys/grantami-recordlists) instead.
+Direct use of this package is unsupported, please use [the PyGranta meta-package](https://grantami.docs.pyansys.com) instead.
```

Comment:
```suggestion
Direct use of this package is unsupported. Use [the PyGranta meta-package](https://grantami.docs.pyansys.com) instead.
```
Tech doc style guides say to omit the word "please".

Reply:
Thanks! I've made this change upstream in the template repository and added you as a reviewer: https://github.com/ansys/openapi-client-template/pull/164